packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -215,7 +215,10 @@ export class InsuredBridgeL1Client {
 
   getBridgePoolForL2Token(l2Token: string, chainId: string): BridgePoolData {
     const bridgePoolData = Object.values(this.bridgePools).find((bridgePool) => {
-      return bridgePool.l2Token[chainId].toLowerCase() === l2Token.toLowerCase();
```

Comment:
Didn't the error say that one of these was _undefined_? Do we know why that happened?

Reply:
Ah I figured it out. The issue is that there are two L2 tokens for one bridgepool, one for arbitrum and one for optimism. This errors out when searching through the bridge pools via `.find()` because calling `bridgePool.l2Token[chainId]` is undefined for the some `bridgePools` such as `BADGER` which doesn't have an entry for the network with chain ID 10. So this PR will fix it because it won't be calling a function on undefined anymore.

Reply:
What happens if you call toChecksumAddress on undefined?

Reply:
returns ''

Reply:
Ye In all cases, whenever dealing with addresses, we must use this syntax. Lower case is not tie way. This is a code smell. We are good about this in the other bots and should try stay consistent

Reply:
Cool cool, sounds good. As long as it doesn't error, SGTM